framework/lib/components/switch/switch-field.tsx

```diff
@@ -4,39 +4,73 @@ import { SwitchFieldProps } from './types'
 import { Field } from '../form'
 import { Switch } from './switch'
 import { Box } from '../box'
+import { Flex } from '../flex'
+import { Label } from '../typography'
 
 /**
  * The switch component wrapped in a <Field />
  * meant to be used only inside <Form />
  * @see switch
  * @see {@link https://northlight.dev/reference/switch-field}
+* @example (Example)
+ * ## Basic
+ * Tip:
+ * * For `labelPosition` you can use `"left" | "right"`
+ * * For `labelSize` you can use `"2xs" | "xs" | "md" | "lg"` or leave it default/blank
+ * (?
+ * <Form initialValues={{name: ''}}>
+ *  <SwitchField
+ *   name="terms"
+ *   label="I agree to the Terms & Conditions"
+ *   labelPosition="right"
+ *   labelSize="md"
+ *  />
+ * </Form>
+ * ?)
  *
  */
+
 export const SwitchField = ({
   name,
   label,
   isRequired,
   validate,
   onChange: onChangeCallback = identity,
   direction = 'row',
+  labelPosition = 'right',
```

Comment:
maybe change to labelPlacement to have more continuity? (iconPlacement in FlipButton)